src/main/java/carpet/script/EntityEventsGroup.java

```diff
@@ -108,6 +109,25 @@ public void addEvent(Event type, ScriptHost host, FunctionValue fun, List<Value>
                 );
             }
         };
+        public static final Event ON_MOVE = new Event("on_move", 3)
```

Comment:
Oh, you made a typo here, it has 4 arguments not 3.

Reply:
Oh that is correct, the argument count does not include the first argument which is the entity for every event.